packages/playground-examples/copy/pt/3-7/Fixits/Infer From Usage Changes.ts

```diff
@@ -0,0 +1,35 @@
+//// { compiler: {  noImplicitAny: false }, order: 2 }
+
+// Na versão 3.7 do TypeScript há existência de 'dedução por uso'
+// deixando o codigo mais inteligênte, Você agora poderá usar
+// uma lita de importantes tipos conhecidos (string, number, array, Promise)
+// e deduzer se o uso do tipo combina com a API de objetos.
+
+// Para o próximos pequenos exmplos, selecione o parâmetro das
+// funções, clique na lâmpada e selecione "Deduzier os tipos de Parâmetros..."
```

Comment:
```suggestion
// Para o próximos pequenos exemplos, selecione o parâmetro das
// funções, clique na lâmpada e selecione _Infer parameter types from usage_ (Deduzir os tipos de parâmetros pelo uso)
```
![image](https://user-images.githubusercontent.com/17657014/90267952-a8320880-de2c-11ea-9564-150cf733dba2.png)